src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const daysForSmallDiscount = 3;
+  const daysForBigDiscount = 7;
+  const carRentPricePerDay = 40;
+  const smallDiscountPercentage = 20;
+  const bigDiscountPercentage = 50;
+
+  if (days < daysForSmallDiscount) {
+    return carRentPricePerDay * days;
```

Comment:
We use `carRentPricePerDay * days` multiple times, maybe it can be moved to a const?